VS_Meadow_Extension/VS_Meadow_Extension.2022/VS_Meadow_Extension.2022.csproj

```diff
@@ -1,240 +1,240 @@
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Mono.Debugging.VisualStudio">
+      <HintPath>$(DevEnvDir)\Extensions\Xamarin.VisualStudio\Mono.Debugging.VisualStudio.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build" Version="15.1.0.0" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="15.1.0.0" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.1.0.0" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.1.0.0" />
+    <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="Microsoft.VisualStudio.CommandBars" Version="8.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" />
+    <PackageReference Include="Microsoft.VisualStudio.Composition" Version="15.8.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Composition.NetFxAttributes" Version="15.8.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.CoreUtility" Version="15.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Designer.Interfaces" Version="1.0.5000.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Imaging" Version="15.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" Version="14.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.OLE.Interop" Version="7.1.40304.0" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem">
+      <Version>16.2.133-pre</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Managed" Version="2.0.1.0" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Managed.VS" Version="2.0.1.0" />
+    <PackageReference Include="Microsoft.VisualStudio.SDK">
+      <Version>17.0.31902.203</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop" Version="7.1.40304.0" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop.10.0" Version="10.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop.11.0" Version="11.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop.12.0" Version="12.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.TextManager.Interop.8.0" Version="8.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.0.64" />
+    <PackageReference Include="Microsoft.VisualStudio.Utilities" Version="15.0.0.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Validation" Version="15.3.0.0" />
+    <PackageReference Include="Microsoft.VSSDK.BuildTools">
+      <Version>17.2.2198</Version>
```

Comment:
So far, this is the only change, though it also bumped a sub-package dependency to align with it.

Reply:
@patridge What was the sub-package?

Reply:
Could the change me isolated, so that the whole csproj isn't re-written? Maybe just 
`<PackageReference Include="Microsoft.VSSDK.BuildTools" Version="17.2.2198" />`
And possibly the sub-package only
?

Reply:
We could possibly do that, but Git was being really mean about the diff. It did the same for the commit that changed it to the prior version from the preview it was on. I wonder if it's some whitespace change happening, like leading stuff or line endings.